### PR TITLE
Update models to match sample data

### LINKED ISSUE #19: Update models based on sample content
FreeFrom provided us with some sample content (check pinned messages in the #proj-free-from Slack). We need to update the existing models to match the slight discrepancies in the sample content.

### PR DESCRIPTION
**Pull request by egiurleo**
I have added some fields to the Resource in correspondence with some sample content that FreeFrom sent us. I have also included a class called ResourceLink, which represents an external link to a form or informational website.

Closes #19

### CODE REVIEW

File: spec/controllers/resources_controller_spec.rb [hunk 1]
```diff
@@ -81,13 +81,19 @@
...
             {
-              state: 'NY',
-              time: 'Months. Note: Depending on the case, it could take longer.',
-              cost: 'It is free of charge.',
-              award: 'You can potentially claim full amount of reasonable losses in awards for shattered computer, ER visits, or lost days from work.',
-              likelihood: 'The likelihood to get reimbursement through this option depends on whether a criminal case is brought. The system takes care of everything but that only happens if the evidence is strong enough for a prosecutor to bring charges.',
-              safety: 'It is likely that the prosecutor will call you to testify in the criminal case with your abuser present.',
-              story: 'You will have to share your story when you make a report to law enforcement and if you are called to testify, you will have to do so on the stand at trial.',
+              state: 'WV',
+              who: 'You are eligible to apply if you were injured as the result of a crime.You are also eligible to apply if you are the legal dependent of someonewho was killed as the result of a crime',
```
Comment (JZenk) [inline spec/controllers/resources_controller_spec.rb]: Needs a space between `crime.` and `You`. Later in the line someonewho should be broken up into someone who.

File: spec/controllers/resources_controller_spec.rb [hunk 2]
```diff
@@ -81,13 +81,19 @@
...
+              when: 'The application must be received within 2 years after the date ofthe crime',
```
Comment (JZenk) [inline spec/controllers/resources_controller_spec.rb]: Space between of and the.

File: spec/controllers/resources_controller_spec.rb [hunk 3]
```diff
@@ -81,13 +81,19 @@
...
+              time: 'On average, it takes 7.5 months for an application to be approvedor denied',
```
Comment (JZenk) [inline spec/controllers/resources_controller_spec.rb]: Space between approved or.

File: spec/controllers/resources_controller_spec.rb [hunk 4]
```diff
@@ -81,13 +81,19 @@
...
+              cost: 'It is free to apply',
+              award: 'If you were injured, the maximum you can receive for your expenses is $35,000.',
+              covered_expenses: 'You can get reimbursed for the following expenses: Medical / dental expenses; funeral and burial expenses; mental healthcounseling; lost wages / income; lost support of eligible dependents; andmileage to medical treatment facilities.',
+              likelihood: 'Your chances of getting reimbursement depend on your meeting all of the application’s criteria. See “How-to File” below for moreinformation.',
```
Comment (JZenk) [inline spec/controllers/resources_controller_spec.rb]: Space in more information.

File: spec/controllers/resources_controller_spec.rb [hunk 5]
```diff
@@ -81,13 +81,19 @@
...
+              safety: 'This application is made directly to the state. You will not need toconfront your harm-doer to apply.',
```
Comment (JZenk) [inline spec/controllers/resources_controller_spec.rb]: Space in to confront.

File: spec/controllers/resources_controller_spec.rb [hunk 6]
```diff
@@ -81,13 +81,19 @@
...
+              story: 'You will have to include information about what happened to you inyour application',
```
Comment (JZenk) [inline spec/controllers/resources_controller_spec.rb]: In your.

File: spec/controllers/resources_controller_spec.rb [hunk 7]
```diff
@@ -81,13 +81,19 @@
...
+              tips: [
+                ' If you are close to the 2 year deadline but don’t have certain documents (police report, invoices, receipts) do not wait to submit yourapplication.   Instead,   submit   the   application   and   follow-up   with   thedocuments when you get them.'
```
Comment (JZenk) [inline spec/controllers/resources_controller_spec.rb]: Spacing is odd.

File: spec/controllers/resources_controller_spec.rb (hunk 1 above)
Comment (egiurleo) [reply spec/controllers/resources_controller_spec.rb]: Thanks for these comments. I'm not sure why the spacing got so messed up.

**Pull request merged**